### DISCUSSION TITLE
Is it possible to call somehow call Microsoft.Office.Interop.Word methods in the browser?

### DISCUSSION THREAD

**Discussion by jaunruh** [unoplatform/Uno.Wasm.Bootstrap#441] [Q&A] [answered]
I know this might be a long shot. But I am trying to call some methods from the `Microsoft.Office.Interop.Word` package in the frontend. I tried to just add these dependencies to the code and execute it. That did not work (as expected). Is there a way to get this to work theoretically? Could an expert answer me this question please?

Thank you very much for your time.

Best regards.

**Comment by jeromelaban** [comment, marked as answer]
The question to ask is generally: Can JavaScript do it as well? If the answer is no, then for WebAssembly it will generally be no as well.

In this case, this particular package likely uses COM interop to communicate with Word, and the browser's sandbox won't allow for it.

**Reply by jaunruh** [reply]
Yes as far as I understand it that package uses COM interop. Security / Sandbox issues were pretty much my assumption. Thank you.
